gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -398,6 +406,8 @@ local position = 0
 local frameCount = 10
 
 local lastStrafeValue = 0
+local lastFOVValue = 85
+local multiplier = 40
```

Comment:
Rename this, so it is clear that it belongs to FOV stuff

Reply:
ok